ansible/handlers/main.yml

```
@@ -13,6 +13,12 @@
     state: "restarted"
   listen: "restart oauth2-proxy"
 
+- name: "restart oauth2-proxy (admin_"
+  service:
+    name: "oauth2-proxy"
```

This would restart the original (non-admin) `oauth2-proxy.service`

```suggestion
    name: "oauth2-proxy-admin"
```